### PR TITLE
chore: upgrade deps

### PR DESCRIPTION
**Pull request by Atinux**
Would love your help on some issues during dev @danielroe 

- An issue with TS Loader
Screenshot 2021-05-12 at 14 03 24

- Some images does not work with latest nuxt/image

### CODE REVIEW

**Comment by danielroe** [comment]
New TS loader depends on webpack 5.

**Comment by danielroe** [comment]
* ts-loader@9 requires webpack 5
* vercel images issues will be fixed with next release of nuxt/image (see https://github.com/nuxt/image/pull/246)

**Comment by Atinux** [comment]
Interesting error:

**Comment by pi0** [comment]
`ts-loader` should be removed from dependencies and tailwind regression is fixed. I think would be easier letting renovate handling dep upgrades :)

File: nuxt.config.ts [hunk 1]
```diff
@@ -46,16 +46,15 @@ export default <NuxtConfig> {
...
     screens: {
       icon: 40,
       avatar: 24
     },
     domains: [
-      'avatars0.githubusercontent.com',
-      'avatars1.githubusercontent.com',
-      'avatars2.githubusercontent.com',
-      'avatars3.githubusercontent.com'
+      'https://avatars0.githubusercontent.com',
```
Comment (pi0) [inline nuxt.config.ts:54]: #277

**Comment by danielroe** [comment]
👍 Should be good to go (and up 5 points on Lighthouse 😁)

**Comment by pi0** [comment]
https://github.com/nuxt/modules/pull/146#issuecomment-842206716

**Pull request closed without merging**